### PR TITLE
Updated project file for Xcode 7

### PR DESCRIPTION
**Pull request by taquitos**
call designated initializer in FABJLTimer.m

### CODE REVIEW

File: JLJSONMapping.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -556,6 +557,7 @@
 				);
 				GCC_PRECOMPILE_PREFIX_HEADER = NO;
 				INFOPLIST_FILE = "JLJSONMappingTests/Supporting Files/JLJSONMappingTests-Info.plist";
+				PRODUCT_BUNDLE_IDENTIFIER = "com.mydoghatestechnology.${PRODUCT_NAME:rfc1034identifier}";
```
Comment (armcknight) [inline JLJSONMapping.xcodeproj/project.pbxproj]: :dog: :-1: :computer:

File: JLJSONMapping/Util/FABJLTimer.m [hunk 2]
```diff
@@ -30,6 +30,11 @@ - (id)initWithStartTimerName:(NSString *)name
     return self;
 }
 
+- (instancetype)init
```
Comment (armcknight) [inline JLJSONMapping/Util/FABJLTimer.m]: Should we add the following to the .h?

``` objective-c
- (instancetype)init NS_UNAVAILABLE;
+ (instancetype)new NS_UNAVAILABLE;
```

**Comment by armcknight** [comment]
Could we go ahead and commit (I think)

```
JLJSONMapping.xcodeproj/project.xcworkspace/contents.xcworkspacedata
```

and ignore

```
JLJSONMapping.xcodeproj/project.xcworkspace/xcuserdata/*
JLJSONMapping.xcodeproj/xcuserdata/*
```

File: JLJSONMapping.xcodeproj/project.pbxproj (hunk 1 above)
Comment (taquitos) [inline JLJSONMapping.xcodeproj/project.pbxproj]: xcode doing it's thing "upgrading" the project

File: JLJSONMapping/Util/FABJLTimer.m (hunk 2 above)
Comment (taquitos) [inline JLJSONMapping/Util/FABJLTimer.m]: yeah

**Comment by taquitos** [comment]
@armcknight added the .gitignore and included contents.xcworkspacedata

**Comment by armcknight** [comment]
Awesome! :rowboat:

**Pull request merged**